index.js

```diff
@@ -886,6 +886,41 @@
     return idx >= 0 && idx < len ? Just(xs[idx]) : Nothing();
   });
 
+  //# slice :: Number -> Number -> [a] -> Maybe [a]
+  //.
+  //. Returns Just a list containing the elements from the supplied list
+  //. from a beginning index (inclusive) to an end index (exclusive).
+  //. Returns Nothing if the range specified by the two indices exceeds
+  //. length of the list. Accepts negative indices, which indicate an
```

Comment:
It's not accurate to write _exceeds length of the list_. Perhaps we could write _Returns Nothing unless the start interval is less than or equal to the end interval, and the list contains both (half-open) intervals._